benefits/settings.py

```diff
@@ -72,6 +72,7 @@ def RUNTIME_ENVIRONMENT():
     "https://www.googleapis.com/auth/userinfo.email",
     "https://www.googleapis.com/auth/userinfo.profile",
 ]
+SSO_SHOW_FORM_ON_ADMIN_PAGE = False
```

Comment:
@angela-tran Should this an environment variable or can we have this False all the time?

Reply:
No need for an environment variable, especially since we'll have the separate view in #2297. What you have looks good!